### ISSUE TITLE
Syntax: canonical form of minus cannot be read

### ISSUE THREAD

**Issue by UWN** [ichiban/prolog#210]
```
?- writeq(-(1)).  
2022/04/24 08:09:21 failed to query: unexpected token: <paren L (> % unexpected ; expected - (1)
?- writeq(-(1,1)).
2022/04/24 08:09:16 failed to query: unexpected token: <paren L (> % unexpected ; expected 1-1
?- writeq(-(1,2,3)). 
2022/04/24 08:11:46 failed to query: unexpected token: <paren L (> % unexpected ; expected -(1,2,3)
```

**Comment by ichiban** [comment]
@UWN The parser/lexer were completely overhauled in #227 and handling of minus was fixed along the way. Spacing is still off but it'll be fixed in #206.

```console
$ git checkout main
Already on 'main'
Your branch is up to date with 'origin/main'.
$ go install github.com/ichiban/prolog/cmd/1pl
$ $(go env GOPATH)/bin/1pl
Top level for ichiban/prolog (devel)
This is for testing purposes only!
See https://github.com/ichiban/prolog for more details.
Type Ctrl-C or 'halt.' to exit.
?- writeq(-(1)).
-1true.
?- writeq(-(1,1)).
1-1true.
?- writeq(-(1,2,3)).
-(1, 2, 3)true.
?- 
```

**Comment by ichiban** [comment]
Fixed in `v0.10.0`.

```console
$ $(go env GOPATH)/bin/1pl
Top level for ichiban/prolog v0.10.0
This is for testing purposes only!
See https://github.com/ichiban/prolog for more details.
Type Ctrl-C or 'halt.' to exit.
?- writeq(-(1)).
- (1)true.
?- writeq(-(1,1)).
1-1true.
?- writeq(-(1,2,3)).
-(1,2,3)true.
```
